### ISSUE TITLE
Update hex logos image on homepage

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [tidyverse/tidyverse.org#212]
The current image on the homepage shows hex logos 6 packages, I think it should show all 8 in core tidyverse.

This would have the additional benefit of getting more 🐱 on the homepage. 🎉

**Comment by hadley** [comment]
Duplicate of #95

**Comment by hadley** [comment]
Looks like this is now controlled by `layout/index.html`.

@apreshill in principle, it looks like I could just add a `col1` section. Is it ok for me to do that in `layout/index.html` or does it need to do upstream in a theme file somewhere?

**Comment by apreshill** [comment]
Hi, 

Here is what would need to change. It could change just in tidyverse.org, but if we go to all the trouble of changing the CSS it would be great to do it in the theme so tidymodels.org could also use it easily.

1. The current rows/columns are defined in the CSS, done I believe by Robby S. We would need new CSS classes for any new hex positions: https://github.com/rstudio/hugo-tourmaline/blob/master/static/css/main-site.css#L1205-L1232

1. We would also need to consider how this will look on mobile: https://github.com/rstudio/hugo-tourmaline/blob/master/static/css/main-site.css#L1234-L1240

1. Once that is done, I parameterized this so you can add them to your `index.html` yaml and the corresponding layout file will need to have new slots added (this is pretty quick!): https://github.com/rstudio/hugo-tourmaline/blob/master/layouts/index.html#L11-L25

**Comment by hadley** [comment]
Re-opening this issue since I think I closed the wrong one.

**Comment by juliasilge** [comment]
With us finally getting a hex for workflows, this would be nice to figure out for tidymodels.org as well.
